packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/OneTimeDepositView.svelte

```diff
@@ -2,18 +2,22 @@
     import { Button, FontWeight, Text, TextType } from 'shared/components'
     import { localize } from '@core/i18n'
     import { setClipboard } from '@core/utils'
+    import { selectedWallet } from '@core/wallet'
+    import { onMount } from 'svelte'
 
     let isVisibleAddress: boolean = false
+    let depositAddress: string = ''
 
-    // TODO: Replace with proper address
-    const DUMMY_ADDRESS = 'iota1q9f0mlq8yxpx2nck8a0slxnzr4ef2ek8f5gqxlzd0wasgp73utryjtzcp98'
+    onMount(async () => {
+        depositAddress = await $selectedWallet?.implicitAccountCreationAddress()
+    })
```

Comment:
Wouldn't it be better to make the `implicitAccountCreationAddress()` request by clicking on `showAddress` instead of `onMount`?

Although it is an asynchronous function, the request will be almost instantaneous so there will not be much difference but, I think it is better to wait when the button is clicked than when the component is mounted